sorting_robot/State location
---
sorting_robot/TrafficLight signal